src/main/cpp/auton/PrimitiveParams.h

```diff
@@ -27,6 +27,7 @@
 #include "auton/PrimitiveEnums.h"
 #include "chassis/ChassisOptionEnums.h"
 #include "DragonVision/DragonVision.h"
+#include "DragonVision/DragonCamera.h"
```

Comment:
Why do we need this added include?

Reply:
This include is for the DragonCamera::PIPELINE enum type.  However, we include DragonCamera in DragonVision so it can be removed.

Reply:
Removed